netty/src/main/java/io/grpc/transport/netty/NettyClientHandler.java

```diff
@@ -241,15 +241,19 @@ public boolean visit(Http2Stream stream) throws Http2Exception {
   }
 
   @Override
+  public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) throws Exception {
```

Comment:
Just so I understand what you're trying to do here ... this is to capture any non HTTP/2 errors in the pipeline? In particular, startup-related errors I'm guessing?